src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -374,6 +374,39 @@ class GrpcLb : public LoadBalancingPolicy {
     PickResult Pick(PickArgs args) override;
 
    private:
+    // A subchannel call tracker that unrefs the GrpcLbClientStats object
+    // in the case where the subchannel call is never actually started,
+    // since the client load reporting filter will not be able to do it
+    // in that case.
+    class SubchannelCallTracker : public SubchannelCallTrackerInterface {
+     public:
+      SubchannelCallTracker(
+          RefCountedPtr<GrpcLbClientStats> client_stats,
+          std::unique_ptr<SubchannelCallTrackerInterface> original_call_tracker)
```

Comment:
It just means that the child policy did not return a subchannel call tracker, in which case there's nothing to delegate to.

In this particular case, I could have not bothered to cover this case at all, because the child of the grpclb policy is always round_robin, and round_robin never returns a subchannel call tracker.  But on general principles, it seems better for parent policies to not make assumptions about what their child policy is going to return, so that we can change child policies later without worrying so much about whether the parent will do the right thing with the result.